ovs/vswitch.go

```diff
@@ -147,6 +147,31 @@ func (v *VSwitchService) GetController(bridge string) (string, error) {
 	return strings.TrimSpace(string(address)), nil
 }
 
+// CreateSFlow configures an sFlow collector for the vSwitch.
+func (v *VSwitchService) CreateSFlow(bridgeName string, agentIP string, collectorIP string, collectorPort string, headerBytes string, samplingN string, pollingSecs string) (string, error) {
+	var (
+		agent    = fmt.Sprintf("agent=%s", agentIP)
+		target   = fmt.Sprintf("target=\"%s:%s\"", collectorIP, collectorPort)
+		header   = fmt.Sprintf("header=%s", headerBytes)
+		sampling = fmt.Sprintf("sampling=%s", samplingN)
+		polling  = fmt.Sprintf("polling=%s", pollingSecs)
+
+		sFlowID = "sflow"
+	)
+
+	output, err := v.exec(
+		"--",
+		fmt.Sprintf("--id=@%s", sFlowID), "create", "sflow", agent, target, header, sampling, polling,
+		"--",
+		"set", "bridge", bridgeName, fmt.Sprintf("sflow=@%s", sFlowID),
+	)
+	if err != nil {
+		return "", err
```

Comment:
if the return value of this method is supposed to be the sflow UUID, wouldn't be better to return nil?

Reply:
An empty string is not a valid value, I defer from using pointers unless I have too.